### PR TITLE
EAPI6-cleanup

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

File: x11-plugins/wmpinboard/wmpinboard-1.0.1.ebuild
Comment (voyageur) [inline x11-plugins/wmpinboard/wmpinboard-1.0.1.ebuild:1]: On phone so I have not checked others, but this drops the only stable version: https://packages.gentoo.org/packages/x11-plugins/wmpinboard

Comment (a17r) [reply x11-plugins/wmpinboard/wmpinboard-1.0.1.ebuild:1]: indeed, fixed that: https://bugs.gentoo.org/913078

**Comment by atupone** [comment]
For me it's ok

**Review by zmedico — APPROVED** [review]
Thanks!

**Pull request merged**